extensions/amp-form/0.1/amp-form.js

```diff
@@ -218,10 +219,12 @@ export class AmpForm {
         requireAmpResponseSourceOrigin: true,
       }).then(response => {
         this.actions_.trigger(this.form_, 'submit-success', null);
+        this.analyticsEvent_('amp-form-submit-success');
```

Comment:
Yeah the plan was to document these once this change is in prod otherwise we'll get lots of issues opened that this isn't working if we update docs right away. I'll send a follow up PR later.